src/wp-admin/includes/network.php

```diff
@@ -136,19 +136,15 @@ function network_step1( $errors = false ) {
 		die();
 	}
 
-	$hostname  = get_clean_basedomain();
+	$hostname = preg_replace( '/(?::80|:443)$/', '', get_clean_basedomain() );
 	$has_ports = strstr( $hostname, ':' );
 	if ( ( false !== $has_ports && ! in_array( $has_ports, array( ':80', ':443' ), true ) ) ) {
-		echo '<div class="error"><p><strong>' . __( 'Error:' ) . '</strong> ' . __( 'You cannot install a network of sites with your server address.' ) . '</p></div>';
+		echo '<div class="notice notice-warning"><p><strong>' . __( 'Warning:' ) . '</strong> ' . __( 'Install a network of sites with your server address is experimental.' ) . '</p></div>';
```

Comment:
"Install a network of sites with your server address is experimental." doesn't quite make sense in English, it should be "Installing" perhaps.

Reply:
Done. Thank you.